1.5-attributes-of-science.md

```diff
@@ -0,0 +1,45 @@
+## 1.5	Defining Features of Science
+
+Merriam Webster defines science and engineering as follows:
+
+**[Science](http://www.merriam-webster.com/dictionary/science)**:  
+> Knowledge about or study of the natural world based on facts learned through experiments and observation. 
+
+The scientific method is the process by which science is conducted, and is based on the elements of: question, hypothesis, prediction, testing and analysis (https://en.wikipedia.org/wiki/Scientific_method).  
+
+However, going back to Galileo's *Two New Sciences*, it has been recognized that even with a strong scientific process, additional community-level components are required to ensure the validity and credibility of science.  These include: 
+
+* Data recording and sharing
+* Reproducibility 
+* External review
+
+Scientific journals have long been central to ensuring that these community needs are met, by recording the data from experiments, sharing the methods of those experiments so they can be reproduced elsewhere, and providing a mechanism for external reviews.  However, as our data sets become larger, and our models more complex, the limits of the scientific paper as the basic unit of trade are increasingly clear.  While we do not challenge the core role that journals should continue to play, we also recognize that a complementary mechanism is needed to further promote these community scientific goals.  
+
```

Comment:
##TODO: insert examples/links to ways other academic communities have overcome this challenge [ i.e. the ArXive ]